apps/pipelines/migrations/0014_set_default_history_mode.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 5.1.2 on 2025-03-21 05:59
+
+from django.db import migrations
+from apps.pipelines.models import PipelineChatHistoryModes
+from apps.pipelines.nodes.nodes import StartNode, EndNode
+
+CHUNK_SIZE = 100
+
+def _set_default_history_mode(apps, schema_editor):
+    Node = apps.get_model('pipelines', 'Node')
+    node_queryset = Node.objects.exclude(type__in=[StartNode.__name__, EndNode.__name__])
+    nodes = []
+    for node in node_queryset.iterator(chunk_size=CHUNK_SIZE):
+        node.params['history_mode'] = PipelineChatHistoryModes.SUMMARIZE
```

Comment:
not all nodes support history mode - this should only apply updates to known node types

Reply:
Ah yes of course, what was I thinking